desktop/src/main.rs

```diff
@@ -216,12 +217,29 @@ fn load_movie(url: &Url, opt: &Opt) -> Result<SwfMovie, Error> {
     Ok(movie)
 }
 
+fn get_max_screen_size(monitors: impl IntoIterator<Item = MonitorHandle>) -> PhysicalSize<u32> {
```

Comment:
Now that `get_max_screen_size` has a single caller, it can be made less generic: Accept an `event_loop: &EventLoop` parameter and call `event_loop.available_monitors()` here.